plugins/com.google.cloud.tools.eclipse.appengine.validation/src/com/google/cloud/tools/eclipse/appengine/validation/JavaServletElement.java

```diff
@@ -17,18 +17,20 @@
 package com.google.cloud.tools.eclipse.appengine.validation;
 
 import org.eclipse.core.resources.IMarker;
+import org.eclipse.wst.validation.internal.provisional.core.IMessage;
 
 /**
  * A blacklisted java servlet element that will receive a servlet marker. 
  */
 public class JavaServletElement extends BannedElement {
 
+  private static final String message = Messages.getString("web.xml.version");
```

Comment:
Since these are absolutely immutable objects and `private static`, should normally capitalize all letters. You could also think about a better name then just `message`, but I don't mind if you actually embed these fields.